Assets/Scripts/Weapons/Blaster.cs

```diff
@@ -0,0 +1,21 @@
+using UnityEngine;
+
+namespace Weapons
+{
+    public class Blaster : Weapon
+    {
+        [SerializeField] private Transform firePosition;
+        [SerializeField] private Bullet bulletPrefab;
+        [SerializeField] private int bulletSpeed;
+        [SerializeField] private int damage;
+    
+        public override void Attack()
+        {
+            Vector3 bulletSpawnPos = firePosition.position;
+            Vector2 direction = transform.right;
+            Bullet bullet = Instantiate(bulletPrefab, bulletSpawnPos, Quaternion.identity);
```

Comment:
Kannst du hier noch ein parent reinpacken? Dann wewrden die Bullets nicht so hässlich in der Szene angezeigt:P